configs/config.yaml

```diff
@@ -0,0 +1,9 @@
+polltimeinterval: 20s    # Values: "ms", "s", "m", "h".
+age: 7d                  # Values: "d" or "w" or "m" or "y".
```

Comment:
In future we should specify criteria as well, like on "namespaces" or pods etc

Reply:
@kahootali we should fix this right now; its important that we have well defined criteria